sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,2 +1,350 @@
+* *help*: The `help` command is used for getting help about `sem` or an existing `sem` command.
+* *init*: The `init` command is used for adding an existing GitHub repository to Semaphore 2.0 for the first time.
+* *version*: The `version` command is used for getting the version of the `sem` utility.
+
+## Resource types
+
+Semaphore 2.0 supports two types of resources: `secrets` buckets and Semaphore
+2.0 projects. Most resource related operations require a resource name.
+
+## Working with organizations
+
+This group of `sem` commands for working with organizations contains the
+`sem connect` and `sem context` commands.
+
+### sem connect
+
+The `sem connect` command allows you to connect to a Semaphore 2.0 organization
+for the first time and requires two command line arguments. The first command
+line argument is the organization name and the second command line argument is
+the authentication token for that organization – the organization must already
+exist. Organizations are created using the web interface of Semaphore 2.0.
+The authentication token depends on the active user.
+
+Once you connect to an organization, you do not need to execute `sem connect`
+for connecting to that organization again – you can connect to any
+organization that you are already a member of using the `sem context` command.
+
+### sem context
+
+The `sem context` command is used for listing the organizations the active
+Semaphore 2.0 user belong to and for changing between organizations.
+
+The `sem context` command can be used with or without any command line
+parameters. If `sem context` is used without any other command line parameters,
+it returns the list of Semaphore 2.0 organizations the active user belongs to.
+When used with a command line argument, which should be a valid organization
+the active user belongs to, `sem context` will change the active organization
+to the given one.
+
+## Working with resources
+
+This group of `sem` commands includes the four most commonly and frequently
+used commands: `sem create`, `sem delete`, `sem describe` and `sem get`. This
+mainly happens because these are the commands that allow you to work with
+resources.
+
+### sem create
+
+The `sem create` command is used for creating new resources and should always
+be followed by the `-f` flag, which should be followed by a valid path to a
+proper YAML file. Currently there exist two types of YAML configuration files
+that can be handled by `sem create`: secrets and projects configuration files.
+
+### sem delete
+
+The `sem delete` command is used for deleting existing resources, which means
+that is used for deleting Semaphore 2.0 projects and entire `secrets` buckets.
+
+Note that when you delete a `secrets` bucket, then that particular `secrets`
+bucket disappears from the active organization for every user that is a member
+of that organization and there is no way to get it back unless you recreate
```

Comment:
It feels that introducing `users` in this description is just introducing confusion. I mean when it's deleted from organization it's gone. It's maybe more useful to mention that projects that are using it might be affected.